data/presets/playground/sledding.json

```diff
@@ -0,0 +1,10 @@
+{
+    "icon": "maki-playground",
+    "geometry": [
+        "point"
+    ],
+    "tags": {
+        "playground": "sledding"
+    },
+    "name": "Play Sledding Hill"
```

Comment:
What’s the difference between a “play” sledding hill and one that’s [mapped as a hill with a sport](https://www.openstreetmap.org/node/5914822978)? Is it the size, or the lack of other playground equipment nearby?

Reply:
I'd say both: examples I found in Germany seem to be smaller than the one you linked and are near playgrounds. For example, [this hill](https://www.openstreetmap.org/way/841788525) is actually inside of a `leisure=playground` area.